src/foundry/client-esm/applications/api/application.d.mts

```diff
@@ -567,8 +632,19 @@ declare class ApplicationV2<
    * Only applicable to window Applications.
    * @param expanded - Set the controls visibility to a specific state.
    *                   Otherwise, the visible state is toggled from its current value
+   * @param options  - Options to configure the toggling behavior
+   * @returns A Promise which resolves once the control expansion animation is complete
    */
-  toggleControls(expanded?: boolean): void;
+  toggleControls(
+    expanded?: boolean,
+    options?: {
+      /**
+       * Animate the controls toggling.
+       * @defaultValue `true`
+       */
+      animate?: boolean;
+    },
```

Comment:
Can you put this into its own interface? It may be one prop but it's still good to do.